### PR TITLE
[Doc Feature][Contextless app auto update][2981346]

### PR DESCRIPTION
**Pull request by v-ypalikila**
Added content for contextless app update.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/apps-upload.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b1d804395be0b21b949ab37dcbd7b4f491c68012/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fapps-upload.md) | 94 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/70624957-0f37-4333-8575-35c71c6fccf9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/deploy-and-publish/apps-upload.md [hunk 1]
```diff
@@ -81,34 +81,46 @@ Teams provides several ways to open apps. For more information, see [access your
...
+
+1. Select **Update now**.
+
+   :::image type="content" source="../../assets/images/publish-app/update-window.png" alt-text="Update Teams app in manage apps.":::
+
+   The app is automatically updated in all the related contexts (chats, channels, and meetings) where the app is installed.
+
+If you have multiple app updates and you want to update all your apps at once, select **Apps** > **Manage your apps** and then select **Update all**.
+
+:::image type="content" source="../../assets/images/publish-app/update-apps-update-all.png" alt-text="Screenshot shows the Update all option in Teams app store.":::
```
Comment (jacobdevera) [inline msteams-platform/concepts/deploy-and-publish/apps-upload.md:109]: Can this section be removed? The **Update all** functionality is not supported in this current release.

File: msteams-platform/assets/images/publish-app/update-available-manage-apps.png
Comment (jacobdevera) [inline msteams-platform/assets/images/publish-app/update-available-manage-apps.png:1]: Could this UI mock be replaced with the following actual screenshot?
update-an-app_manage-your-apps-2

File: msteams-platform/assets/images/publish-app/update-available.png
Comment (jacobdevera) [inline msteams-platform/assets/images/publish-app/update-available.png:1]: Could this UI mock be replaced with the following actual screenshot?
update-an-app_manage-your-apps-3

File: msteams-platform/assets/images/publish-app/update-window.png
Comment (jacobdevera) [inline msteams-platform/assets/images/publish-app/update-window.png:1]: Could this UI mock be replaced with the following actual screenshot?

update-an-app_ALL-4

File: msteams-platform/assets/images/publish-app/update-apps-update-all.png
Comment (jacobdevera) [inline msteams-platform/assets/images/publish-app/update-apps-update-all.png:1]: **Update all** is currently not supported, so this image can be removed

**Review by jacobdevera — COMMENTED** [review]
Need to update a few screenshots and remove unimplemented functionality. Other than that, looks good.

File: msteams-platform/concepts/deploy-and-publish/apps-upload.md (hunk 1 above)
Comment (v-ypalikila) [reply msteams-platform/concepts/deploy-and-publish/apps-upload.md:109]: Removed update all functionality.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/apps-upload.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9a7c316129fe805d0d4369c61d6d0fed9f41d42d/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fapps-upload.md) | 95 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c1b74975-4af6-403d-83a8-cc667f7ac565) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/apps-upload.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0c9e38bd588d1294d55e3ce3e4abfd5889fc4ce7/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fapps-upload.md) | 95 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/060a7776-93af-464c-9406-7ecc7c1f9c62) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/assets/images/publish-app/update-available-manage-apps.png
Comment (v-ypalikila) [reply msteams-platform/assets/images/publish-app/update-available-manage-apps.png:1]: Updated.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/apps-upload.md](https://github.com/MicrosoftDocs/msteams-docs/blob/74477aa83027b63fbf52470b7f57548e5f935d96/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fapps-upload.md) | 95 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/152e4754-e9cd-4d9f-95bb-2c08d1c24233) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/apps-upload.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e1a4fda4487197255f9e73f679eec75b20f0a9cf/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fapps-upload.md) | 95 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/590d5c99-2b18-4dd5-aa41-40af2c5cdbca) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/apps-upload.md](https://github.com/MicrosoftDocs/msteams-docs/blob/500059e8e91217d34388af380f7dae1d66951184/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fapps-upload.md) | 95 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/810bb89b-9c43-40bc-b873-bbd9630fd6d3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/assets/images/publish-app/update-window.png
Comment (v-ypalikila) [reply msteams-platform/assets/images/publish-app/update-window.png:1]: updated the image.

File: msteams-platform/assets/images/publish-app/update-available.png
Comment (v-ypalikila) [reply msteams-platform/assets/images/publish-app/update-available.png:1]: updated the image.

**Review by jacobdevera — DISMISSED** [review]
LGTM

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/apps-upload.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b701e574141f4c2c740aabad6596c3e1ff60a026/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fapps-upload.md) | 95 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/50fcec59-67c6-4f03-bd2e-9c33e8d6034b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/b701e574141f4c2c740aabad6596c3e1ff60a026/msteams-platform%2Fmstdd-landing.yml) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1bb1ca4b-68fd-4d5f-b38b-fafb4e36fdbf) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b701e574141f4c2c740aabad6596c3e1ff60a026/msteams-platform%2Fwhats-new.md) | 94 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/64a6adaf-56c9-4da4-9ead-342415517fca) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by shmundra20 — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/apps-upload.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f4568da80e7cc2fed8690aec0ab6f1a48272a0a9/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fapps-upload.md) | 95 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cdd3b889-03c5-4d2e-9504-cdaa09165d0f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/f4568da80e7cc2fed8690aec0ab6f1a48272a0a9/msteams-platform%2Fmstdd-landing.yml) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8e099e1a-d574-4df4-8a97-294d91c3bf93) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f4568da80e7cc2fed8690aec0ab6f1a48272a0a9/msteams-platform%2Fwhats-new.md) | 94 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1ca048c7-f488-4c15-b1c3-ce10755c27f5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/apps-upload.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0aa19dd64c1cc93690193408fa9d249c65bbaa83/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fapps-upload.md) | 95 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bc9d6ae8-9276-49ce-8e9a-abde188f6a33) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/0aa19dd64c1cc93690193408fa9d249c65bbaa83/msteams-platform%2Fmstdd-landing.yml) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dc4e10b1-7a32-4bf1-b68a-910488c8d389) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0aa19dd64c1cc93690193408fa9d249c65bbaa83/msteams-platform%2Fwhats-new.md) | 94 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9cb311c6-85d3-44d5-a6cf-e54d65d5c751) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/apps-upload.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b580437c49298baebd26b34d863e036cb2d91c7e/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fapps-upload.md) | 95 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9e12e2cc-9079-4088-ad0d-1bc74b96a035) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/b580437c49298baebd26b34d863e036cb2d91c7e/msteams-platform%2Fmstdd-landing.yml) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e97121b3-4c0e-4e81-aa34-40e87a76c12a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b580437c49298baebd26b34d863e036cb2d91c7e/msteams-platform%2Fwhats-new.md) | 94 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e61a5861-5ca7-4523-9d40-8f24bc099061) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Review by v-habibaumme — APPROVED** [review]

**Pull request merged**